.gitignore

```diff
@@ -1,9 +1,15 @@
+# This file specifies intentionally untracked files that Git should ignore.
+# Files already tracked by Git are not affected.
+# See: https://git-scm.com/docs/gitignore
+
+## Project Specific ##
+findings-data-import.zip
+
+## Python ##
 __pycache__
 .coverage
 .DS_Store
```

Comment:
We can get rid of this file:
```suggestion
```

Reply:
Ack I missed that when doing this one. Thanks for noticing!

Reply:
Force-pushed to integrate it with the merge. Thanks again!